mail/dovecot/Portfile

```diff
@@ -63,16 +64,23 @@ subport dovecot-sieve {
     homepage        https://pigeonhole.dovecot.org/
 
     distname        dovecot-${core_branch}-pigeonhole-${version}
-    master_sites    ${homepage}releases/${core_branch}/
```

Comment:
The switch to using the actual release tarball was quite deliberate, precisely to avoid all the extra complexity of downloading the extra distfiles.

Reply:
The distfiles are necessary to be able to run `autoconf`, which is necessary to patch the code for APNS.

Reply:
The files are all included in the release tarball, so that shouldn't be a problem.